Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1209,9 +1249,14 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
 
   auto caloPHOSCells = recoData.getPHOSCells();
   auto caloPHOSCellsTRGR = recoData.getPHOSTriggers();
-
   auto ctpDigits = recoData.getCTPDigits();
+  const auto& tinfo = pc.services().get<o2::framework::TimingInfo>();
 
+  if (mCTPReadout == 1) {
+    std::vector<o2::ctp::CTPDigit> ctpDigitsCreated;
+    createCTPReadout(recoData, ctpDigitsCreated, tinfo.runNumber);
+    ctpDigits = ctpDigitsCreated;
```

Comment:
`ctpDigitsCreated` is deleted once you go outside `{}` so the span ctpDigits will be invalid. You should move it before the `if`.